lib/cocoapods/project.rb

```diff
@@ -179,13 +179,13 @@ def pod_support_files_group(pod_name, dir)
     # @return [PBXFileReference] The new file reference.
     #
     def add_file_reference(absolute_path, group, reflect_file_system_structure = false)
-      file_path_name = Pathname.new(absolute_path)
+      file_path_name = absolute_path.is_a?(Pathname) ? absolute_path : Pathname.new(absolute_path)
```

Comment:
It's not really, but it made stubbing things on a specific `Pathname` instance possible :)